target/linux/qualcommax/image/ipq807x.mk

```diff
@@ -61,6 +61,21 @@ define Device/buffalo_wxr-5950ax12
 endef
 TARGET_DEVICES += buffalo_wxr-5950ax12
 
+define Device/cmcc_rm2-6
+	$(call Device/FitImage)
+	$(call Device/UbiFit)
+	DEVICE_VENDOR := CMCC
+	DEVICE_MODEL := RM2-6
+	BLOCKSIZE := 128k
+	PAGESIZE := 2048
+	DEVICE_DTS_CONFIG := config@ac02
+	SOC := ipq8070
+	IMAGES := factory.bin sysupgrade.bin
```

Comment:
`IMAGES += factory.bin` to not drop the default `factory.ubi` image

Reply:
But for this device, factory.ubi is useless and can cause confusion.

Reply:
Ok, if that's the case

Reply:
You can remove the call:
```
$(call Device/UbiFit)
```
and add `KERNEL_IN_UBI := 1` if the kernel inside ubi is used.